### ISSUE TITLE
Double Skip or Take query throw exception

### ISSUE THREAD

**Issue by KopIlya** [JasperFx/marten#261]
```
using (var session = store.OpenSession()) {
    var result = session.Query<T>().Skip(4).Skip(4).ToList();
    var result = session.Query<T>().Take(4).Take(4).ToList();
}
```
Both of this requests throw exception: At least one object must implement IComparable.

**Comment by jeremydmiller** [comment]
My first question is "why would you *want* to do that?"

**Comment by KopIlya** [comment]
I get linq query from Client. Query can include Skip or Take method. 
I don't know included in query Take or Skip methods.
I add to query limit 1000 object by Take(1000). If Client add Take method in his query, in this case i have request with double Take methods.

**Comment by jeremydmiller** [comment]
Okay. I found the problem and fixed it. The fix will be in v0.8.6 later today w/ some other fixes.
